tests/StacktraceTest.php

```diff
@@ -80,10 +84,11 @@ public function testAddFrame(): void
 
         $frames = $stacktrace->getFrames();
 
-        $this->assertCount(3, $frames);
-        $this->assertFrameEquals($frames[0], 'TestClass::test_function', 'path/to/file', 12);
-        $this->assertFrameEquals($frames[1], 'test_function', 'path/to/file', 12);
+        $this->assertCount(4, $frames);
+        $this->assertFrameEquals($frames[0], 'CrashyClass', 'path/to/file', 12);
```

Comment:
Please use `TestClass` as class name to be coherent with the rest of the code

Reply:
Renamed.